web/pdf_page_view.js

```diff
@@ -148,16 +334,19 @@ class PDFPageView {
     regularAnnotations: true,
   };
 
-  #layers = [null, null, null, null];
+  #originalViewport = null;
+
+  #layers = [null, null, null, null, null];
```

Comment:
Is this change actually needed now, given that the next patch was updated to place the detailView in the existing `canvasWrapper`?